hail/python/hailtop/hailctl/dev/benchmark/utils.py

```diff
@@ -112,10 +112,21 @@ def run_all():
         _run(benchmark, _n_iter)
 
 
+def run_pattern(pat):
+    _ensure_initialized()
+    test_run = False
+    for name, benchmark in _registry.items():
+        if pat in name:
+            test_run = True
+            _run(benchmark, _n_iter)
+    if not test_run:
+        raise ValueError(f'pattern {repr(pat)} matched no benchmarks')
```

Comment:
not enough to block on this alone. but `{pat!r}` is preferred f-string syntax for repr.